### DISCUSSION TITLE
Running the project on my local machine

### DISCUSSION THREAD

**Discussion by AnirudhPanda** [Praful932/Kitabe#87] [General]
Getting this error message in the bash after running python manage.py migrate after setting the secret key

![Screenshot (116)](https://user-images.githubusercontent.com/66218496/111038422-cad1e180-844e-11eb-9c66-4c68ca46a7f8.png)

Sorry for the silly doubts!!!

**Comment by maneprajakta** [comment]
@AnirudhPanda Line 25 should be : 
`SECRET_KEY = "12345"`

**Reply by AnirudhPanda** [reply]
Thankyou!
